app/src/main/res/layout/activity_yo.xml

```diff
@@ -1,78 +1,97 @@
-                      android:textColor="#FFFFFF"
-                      android:textSize="30sp"
-                      android:textStyle="bold"
-                      android:gravity="center_horizontal"
-                      android:paddingTop="5dp"/>
-
-        </android.support.v7.widget.Toolbar>
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+    <android.support.v7.widget.Toolbar android:layout_width="match_parent"
+                                       android:layout_height="wrap_content"
+                                       android:background="@color/colorPrimary"
+                                       android:id="@+id/action_toolbar"
+                                       android:paddingBottom="10dp"
+                                       android:paddingTop="10dp"
+                                       app:layout_constraintRight_toRightOf="parent"
+                                       app:layout_constraintLeft_toLeftOf="parent"
+                                       app:layout_constraintTop_toTopOf="parent"
+                                       android:minHeight="0dp"
+                                       android:elevation="10dp">
+
+        <TextView android:layout_width="wrap_content"
+                  android:layout_height="wrap_content"
+                  android:layout_gravity="center"
+                  android:textAlignment="center"
+                  android:id="@+id/yo_action_bar_title"
+                  android:text="@string/yo_ching"
+                  android:textColor="#FFFFFF"
+                  android:textSize="30sp"
+                  android:textStyle="bold"
+                  android:gravity="center_horizontal"
+                  android:paddingTop="5dp" />
+
+    </android.support.v7.widget.Toolbar>
+
+
+    <android.support.v4.widget.DrawerLayout
+
+            android:id="@+id/drawerLayout"
+            android:layout_height="match_parent"
+            android:layout_width="match_parent">
+
+        <!--MAIN CONTENT-->
+
+        <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
 
+            <FrameLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintRight_toRightOf="parent"
+                    app:layout_constraintLeft_toLeftOf="parent"
+                    android:layout_marginTop="0dp"
+                    app:layout_constraintTop_toBottomOf="@+id/action_toolbar"
+                    android:layout_marginLeft="0dp"
+                    android:layout_marginRight="0dp"
+                    android:layout_marginBottom="0dp"
+                    android:id="@+id/fragment_container">
+
+                <!--<fragment-->
```

Comment:
Just remove the fragment, don't leave it commented out. 
![](https://render.bitstrips.com/v2/cpanel/9936476-288439948_4-s1-v1.png?transparent=1&palette=1)